tests/test_parse_affected.py

```diff
@@ -45,7 +45,11 @@ def main():
 
         for rule_dir in ssg.rules.find_rule_dirs(guide_dir):
             rule_path = os.path.join(rule_dir, "rule.yml")
-            rule = ssg.build_yaml.Rule.from_yaml(rule_path, env_yaml)
+            try:
+                rule = ssg.build_yaml.Rule.from_yaml(rule_path, env_yaml)
+            except ssg.build_yaml.DocumentationNotComplete:
+                # Happens on non-debug build when a rule is "documentation-incomplete"
+                continue
```

Comment:
For posterity, while merging `stabilization-v0.1.47` into `master`, there was a conflict here.
They come from #4945 and #4962.